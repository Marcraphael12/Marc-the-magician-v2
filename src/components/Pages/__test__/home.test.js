import Home from '../home';
import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';


describe('Home page of the App', () => {
  test('Builds the snapchot of the Home component safely', () => {
    const rend = renderer.create(<Home />);
    expect(rend.toJSON()).toMatchSnapshot();
  });
  test('Render Home component', () => {
    render(<Home />);
  })
});
