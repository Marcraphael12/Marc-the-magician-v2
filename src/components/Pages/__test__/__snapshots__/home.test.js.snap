// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App component 1`] = `
<div
  className="Home"
>
  <h1>
    Welcome to The Math Magic App
  </h1>
  <div>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum,
    </p>
  </div>
</div>
`;

exports[`App renders the Home component safel component 1`] = `
<div
  className="Home"
>
  <h1>
    Welcome to The Math Magic App
  </h1>
  <div>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum,
    </p>
  </div>
</div>
`;

exports[`Home page of the App Builds the snapchot of the Home component safely 1`] = `
<div
  className="Home"
>
  <h1>
    Welcome to The Math Magic App
  </h1>
  <div>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum,
    </p>
  </div>
</div>
`;

exports[`Home page of the App renders the Home component safely 1`] = `
<div
  className="Home"
>
  <h1>
    Welcome to The Math Magic App
  </h1>
  <div>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum, nisl nisi tincidunt nisi, eget egestas nisl nisi euismod nisi. Nullam euismod, nisi vel consectetur interdum,
    </p>
  </div>
</div>
`;
